packages/phone-number-privacy/combiner/src/utils/firebase-configs.ts

```diff
@@ -41,6 +41,8 @@ export const pnpFullNodeRetryCount = defineInt('PNP_FULL_NODE_RETRY_COUNT', {
 export const pnpFullNodeDelaysMs = defineInt('PNP_FULL_NODE_DELAY_MS', {
   default: RETRY_DELAY_IN_MS,
 })
+export const pnpShouldMockAccountService = defineBoolean('PNP_SHOULD_MOCK_ACCOUNT_SERVICE')
+export const pnpMockDek = defineString('PNP_MOCK_DECK')
```

Comment:
could set a default DEK based on [this](https://github.com/celo-org/celo-monorepo/blob/b4c575e0ae64a05329a8dc65363dd1277a78acb2/packages/phone-number-privacy/combiner/test/end-to-end/resources.ts#L87)

Reply:
Nice, I didn't know that. Adding it

Reply:
Mmm, I'll leave it like this. If I include that function I'll need to add a file in a test folder, which could be a problem for productive builds, or move that to some sort of utils, but just for testing.
If we use it as e2e we can set this variable and that's it